init-tools.sh

```diff
@@ -64,6 +64,9 @@ get_current_linux_name() {
 }
 
 if [ -z "$__DOTNET_PKG" ]; then
+    if [ "$(uname -m | grep "i[3456]86")" = "i686" ]; then
```

Comment:
I copied it from corefx/init-tools.sh. I don't know anything about Docker, Linux x86 and anything else